docs/CreateEKSClusterForScalarDBCluster.md

```diff
@@ -0,0 +1,163 @@
+# Guidelines for creating an EKS cluster for ScalarDB Cluster
+
+This document explains the requirements and recommendations for creating an Amazon Elastic Kubernetes Service (EKS) cluster for ScalarDB Cluster deployment. For details on how to deploy ScalarDB Cluster on an EKS cluster, see [Deploy ScalarDB Cluster on Amazon EKS](./ManualDeploymentGuideScalarDBClusterOnEKS.md).
+
+## Before you begin
+
+You must create an EKS cluster based on the following requirements, recommendations, and your project's requirements. For specific details about how to create an EKS cluster, see the official Amazon documentation at [Creating an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+
+## Requirements
+
+When deploying ScalarDB Cluster, you must:
+
+* Create the EKS cluster by using Kubernetes version 1.21 or higher.
+* Configure the EKS cluster based on the version of Kubernetes and your project's requirements.
+
+## Recommendations (optional)
+
+The following are some recommendations for deploying ScalarDB Cluster. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
+
+### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Cluster node group
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Cluster pod and one Envoy pod on one worker node. Note that the Envoy pod does not exist when you use `direct-kubernetes` mode.
```

Comment:
This part is a bit different from ScalarDB Server. ScalarDB Cluster has two client modes. If you use `direct-kubernetes` mode, you don't need to deploy Envoy.